kolibri/logger/test/test_api.py

```diff
@@ -387,6 +387,11 @@ def setUp(self):
         self.user2 = FacilityUserFactory.create(facility=self.facility)
         self.exam = Exam.objects.create(title="", channel_id="", question_count=0, collection=self.facility, creator=self.user2, active=True)
```

Comment:
If you can, could you improve this test by making `question_count = 1`, then adjusting your `ExamAttempLogs` accordingly?